apps/website/package.json

```diff
@@ -27,14 +27,16 @@
     "@types/serve-static": "^1.15.5",
     "happy-dom": "^12.10.3",
     "start-server-and-test": "^2.0.3",
+    "tsx": "^4.7.1",
     "vitest": "^1.1.1"
   },
   "scripts": {
     "test:static": "tsc --noEmit && eslint '**/*.{ts,tsx,js,jsx}' --ignore-path='./.gitignore'",
     "full-rebuild": "pnpm clean && pnpm build:waku",
     "start:drupal": "pnpm run --filter @custom/cms start",
     "build:drupal": "CLOUDINARY_CLOUDNAME=test pnpm start-test start:drupal 8888 build:waku",
-    "build:waku": "waku build",
+    "build:waku": "waku build && pnpm redirects",
+    "redirects": "tsx ./src/build/redirects.ts --dir=dist/public",
```

Comment:
ULTRA NITPICK! 🤓 
Maybe call this `build:redirects` to make clear this is part of the build process.

Reply:
This is now renamed, I also think it makes more sense to have it called "build:redirects"